docs/src/examples/components/Form/Types/FormExample.shorthand.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Form, Input, Button } from '@stardust-ui/react'
+
+const fields = [
+  {
+    label: 'First name',
+    control: { as: Input },
```

Comment:
could we provide either `Input` or `input` as a default value - for the sake of it not being necessary for the client to always specify the underlying type?

Reply:
We can provide the control to be by default `{as: Input}` but if the user wants to override it, would have to specify the as again, otherwise we would need to handle all different types of control (object, component etc) Would this be enough?